gokart/target.py

```diff
@@ -135,10 +138,10 @@ def _load(self) -> Any:
         self._remove_temporary_directory()
         return model
 
-    def _dump(self, obj) -> None:
+    def _dump(self, obj, task_params: Optional[dict[Any, str]] = None) -> None:
```

Comment:
accding to `to_str_params` function, it will be dict[str, str]
```suggestion
    def _dump(self, obj, task_params: Optional[dict[str, str]] = None) -> None:
```

Reply:
https://luigi.readthedocs.io/en/stable/api/luigi.task.html#luigi.task.Task.to_str_params

Reply:
Ahh, thank you for suggestion.
As you showed, `to_str_params` returns `dict[str, str]`.